content/docs/hooks-overview.md

```diff
@@ -1,107 +1,107 @@
+Sintaksis [*array destructuring*](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring) memungkinkan untuk memberi nama yang berbeda pada variabel *state* yang dideklarasikan dengan memanggil `useState`. Nama-nama tersebut bukan bagian dari API `useState`. React mengasumsikan jika Anda memanggil `useState` berkali-kali, Anda melakukannya dalam urutan yang sama saat setiap *render*. Kami akan kembali membahas kenapa ini berfungsi dan kapan ini berguna kedepannya.
 
-#### But what is a Hook? {#but-what-is-a-hook}
+#### Tetapi Apa itu Hook? {#but-what-is-a-hook}
 
-Hooks are functions that let you “hook into” React state and lifecycle features from function components. Hooks don't work inside classes -- they let you use React without classes. (We [don't recommend](/docs/hooks-intro.html#gradual-adoption-strategy) rewriting your existing components overnight but you can start using Hooks in the new ones if you'd like.)
+*Hooks* merupakan fungsi yang memungkinkan untuk "mengait" pada fitur-fitur *state* dan siklus React dari komponen fungsi. *Hooks* tidak dapat berfungsi didalam kelas -- *Hooks* memungkinkan menggunakan React tanpa kelas. (Kami [tidak merekomendasikan](/docs/hooks-intro.html#gradual-adoption-strategy) membuat ulang komponen yang sudah ada tetapi Anda dapat menggunakan *Hooks* untuk komponen yang baru jika diinginkan.)
 
-React provides a few built-in Hooks like `useState`. You can also create your own Hooks to reuse stateful behavior between different components. We'll look at the built-in Hooks first.
+React menyediakan beberapa *Hooks* bawaan seperti `useState`. Anda dapat membuat *Hooks* sendiri untuk menggunakan ulang *stateful behavior* antara komponen yang berbeda. Kami akan membahas *Hooks* bawaan terlebih dahulu.
 
->Detailed Explanation
+>Penjelasan Detail
 >
->You can learn more about the State Hook on a dedicated page: [Using the State Hook](/docs/hooks-state.html).
+>Anda dapat belajar lebih lanjut mengenai *Hook* State pada laman khusus: [Menggunakan Hook State](/docs/hooks-state.html).
 
-## ⚡️ Effect Hook {#effect-hook}
+## ⚡️ Hook Effect {#effect-hook}
 
-You've likely performed data fetching, subscriptions, or manually changing the DOM from React components before. We call these operations "side effects" (or "effects" for short) because they can affect other components and can't be done during rendering.
+Anda tentunya pernah melakukan pengambilan data, proses berlangganan atau menerima data secara terus menerus (*subscriptions*), atau secara manual merubah DOM dari komponen React sebelumnya. Kami menyebut operasi-operasi tersebut "*side effects*" atau "efek samping" karena dapat mempengaruhi komponen lain dan tidak dapat dilakukan pada saat proses *render*.
```

Comment:
```suggestion
Anda tentunya pernah melakukan pengambilan data,  berlangganan data (*subscription*), atau secara manual mengubah DOM dari komponen React sebelumnya. Kami menyebut operasi-operasi seperti ini "efek samping (*side effects*)" (atau singkatnya "efek (*effects*)")  karena dapat mempengaruhi komponen lain dan tidak dapat dilakukan pada saat proses *render*.
```

`mengubah` is the correct form according to KBBI
https://kbbi.kemdikbud.go.id/entri/mengubah

I also noticed inconsistency between this section and the next section, here we kept `side effect` as it is while in the next section we translated `side effect` as `efek samping`, I personally prefer them to be translated and its English term added the first time we mention it (`efek samping(*side effect*)`)

What do you think? 😄